package.json

```diff
@@ -6,16 +6,27 @@
   "dependencies": {
     "emotion": "^9.2.4",
     "emotion-server": "^9.2.4",
+    "emotions": "0.0.1",
     "gatsby": "^1.9.277",
     "gatsby-link": "^1.6.45",
     "gatsby-plugin-emotion": "^1.1.17",
     "gatsby-plugin-react-helmet": "^2.0.10",
     "gatsby-plugin-typescript": "^1.4.20",
     "lodash": "^4.17.10",
+    "react": "^15.6.2",
+    "react-dom": "^15.6.2",
     "react-emotion": "^9.2.4",
     "react-helmet": "^5.2.0",
     "typescript": "^2.9.2"
   },
+  "jest": {
```

Comment:
There's already a `jest.config.js`; use that rather than the `package.json` field.